src/expand.rs

```diff
@@ -128,29 +128,22 @@ where
 
     let mut failures = 0;
     for test in tests {
-        let file_stem = test
-            .test
-            .file_stem()
-            .expect("no file stem")
-            .to_string_lossy()
-            .into_owned();
+        let path = test.test.display();
+        let expanded_path = test.test.with_extension("expanded.rs");
```

Comment:
Can you tweak the `EXPANDED_RS_SUFFIX` constant to have this value (without the starting dot) and then put the constant here instead?

```suggestion
        let expanded_path = test.test.with_extension(EXPANDED_RS_SUFFIX);
```

Reply:
I like this particular change a lot, it's much cleaner!